modules/minimum-required-permissions-ipi-gcp-xpn.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-account.adoc
+
+[id="minimum-required-permissions-ipi-gcp-xpn_{context}"]
+= Required GCP permissions for Shared VPC Installations
+
+When you are installing a cluster to a link:https://cloud.google.com/vpc/docs/shared-vpc[shared VPC] you must configure the service account for both the host project and the service project. If you are not installing to a shared VPC, you can skip this section.
+
+The standard installation requirements, listed above, should be applied to the service project. In addition, the host project should apply one of the following configurations to the service account:
+
+.Required permissions for creating firewalls in the host project
+[%collapsible]
+====
+* `projects/<host-project>/roles/dns.networks.bindPrivateDNSZone`
```

Comment:
This `bindPrivateDNSZone` requirement will become optional once we land a bug fix. At the moment it is required in both scenarios, but soon we will allow users to skip this permission and provide their own DNS zone. We should keep this in mind if considering different ways of formatting these sections.